silkworm/silkrpc/core/state_reader.cpp

```diff
@@ -28,7 +28,7 @@
 
 namespace silkworm::rpc {
 
-Task<std::optional<silkworm::Account>> StateReader::read_account(const evmc::address& address, uint64_t block_number) const {
+Task<std::optional<silkworm::Account>> StateReader::read_account(const evmc::address& address, BlockNum block_number) const {
```

Comment:
The same change here and in the rest of functions in the module should be applied also in `state_reader.hpp`.

Reply:
ok